site/_docs/history.md

```diff
@@ -132,7 +132,7 @@ Taking `RexBuilder#makeIn($0, ARRAY [100, 200], ARRAY [300, 400])` as an example
 Before: `OR(=($0, ARRAY(100, 200)), =($0, ARRAY(300, 400)))`
 After: `SEARCH($0, Sarg[[100:INTEGER, 200:INTEGER]:INTEGER NOT NULL ARRAY, [300:INTEGER, 400:INTEGER]:INTEGER NOT NULL ARRAY]:INTEGER NOT NULL ARRAY)`
 
-* [<a href="https://issues.apache.org/jira/browse/CALCITE-6959">CALCITE-6959</a>]
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6959">CALCITE-6959</a>] Support `LogicalAsofJoin` in `RelShuttle`.
 * [<a href="https://issues.apache.org/jira/browse/CALCITE-6961">CALCITE-6961</a>]
 Addition of non-default methods for `LogicalAsofJoin` and `LogicalRepeatUnion`, respectively, to `RelShuttle`.
```

Comment:
The description here combines two ISSUES, so there is no problem.

Reply:
Do we split the two descriptions, or fit the two CALCITE-X into one line? Because it looks strange on the website, 6959 has no description at all, but 6961 has accomplished two things. If you think the current format is okay, then I will close this PR first.